### ISSUE TITLE
On cards, it is not clear where the user has to click to open the content of the card.

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#950]
On cards, whether organizations, projects, or individuals it is not clear where user has to click to expand/open the card content. The heading of the card is the link but it doesn't have link styling. It is suggested to add an underline under the heading or make the whole card clickable.
